src/runtime/utils/colors.ts

```diff
@@ -1,4 +1,4 @@
-import { omit } from './runtime/utils/lodash'
+import { omit } from './lodash'
```

Comment:
Why move this to runtime?

Reply:
The config template makes use of it https://github.com/nuxt/ui/pull/1272/files#diff-32977ac9b4dda8ed881a84dff6072d1511082d391dee1a72bb685cfcd9f5be75R87